tensorflow/tools/xla_extract/tf_graph_to_xla_lib.cc

```diff
@@ -287,6 +301,13 @@ Status xla_extract_via_strings(const std::string& graph_def_msg,
   auto hmod = ExtractHloFromGraphDef(gdef, target_node);
   hmod.SerializeToString(out_graph);
 
+  char* value = std::getenv("XLA_LOG");
+  char default_val = '0';
+  int xla_log = atoi(value ? value : &default_val);
```

Comment:
why not 
```cpp
int xla_log = value ? atoi(value) : 0;
```
rather than the more expensive atoi("0")?
Also, I don;t think &default_val guarantees a null terminator, so that behavior is actually probably undefined to some degree.

Reply:
can replace it with `int xla_log = value ? atoi(value) : 0;`